recipes/seiqrdp-model/meta.yaml

```
@@ -0,0 +1,52 @@
+package:
+  name: seiqrdp-model
+  version: 7.5.3
+
+source:
+  url: https://pypi.io/packages/source/s/seiqrdp-model/seiqrdp_model-7.5.3.tar.gz
```

It's better to use a version variable to make sure the version and sources are kept in sync. It also makes it possible to the bot to do automatic update PRs:
```suggestion
  url: https://pypi.io/packages/source/s/seiqrdp-model/seiqrdp_model-{{ version }}.tar.gz
```